crates/oxc_linter/src/builder.rs

```diff
@@ -421,9 +421,16 @@ impl RulesCache {
         let mut all_rules: Vec<_> = if self.plugins.is_all() {
             RULES.clone()
         } else {
+            let mut plugins = self.plugins;
+
+            // we need to include some jest rules when vitest is enabled, see [`VITEST_COMPATIBLE_JEST_RULES`]
+            if plugins.contains(LintPlugins::VITEST) {
+                plugins = plugins.union(LintPlugins::JEST);
+            }
```

Comment:
would this enable the default jest rules?

Reply:
I do not think. This is only the `RulesCache` where the Linter will pick available rules. 
Maybe someone with more knowledge can answer this :)